app/src/views/features/sessions/SessionViewer/SessionDetails.tsx

```diff
@@ -103,7 +103,10 @@ const SessionDetails: React.FC = () => {
     }
 
     player.goto(startTimeOffset * 1000, true);
-  }, [player, startTimeOffset]);
+    // player should start playing from the start time offset only on the
+    // first load and not when the user changes time offset.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [player]);
```

Comment:
Consider a case where player was at 5sec and I selected offset to 20sec. I would like to see the frame at 20sec. 
In case we don't want this, skipping dependency in `useEffect` is not the right way. We can add another state to component to keep track the player was initialized. 

Anyway, I think performance degradation is because of another issue. I see in `SessionPropertiesPanel` that we don't update redux store with debounce, so it triggers render cycle on every character stroke in the offset input.